packages/flutter/test/material/page_test.dart

```diff
@@ -785,4 +786,34 @@ void main() {
     expect(homeTapCount, 2);
     expect(pageTapCount, 1);
   });
+
+  testWidgets(
+      'On iOS, a MaterialPageRoute should slide out with CupertinoPageTransition'
```

Comment:
The indentation seems off, we use 2 spaces instead of 4. See: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#indent-multi-line-argument-and-parameter-lists-by-2-characters

Reply:
fixed!